nucypher/datastore/datastore.py

```diff
@@ -84,7 +84,7 @@ class Datastore:
 
     # LMDB has a `map_size` arg that caps the total size of the database.
     # We can set this arbitrarily high (1TB) to prevent any run-time crashes.
-    LMDB_MAP_SIZE = 1_000_000_000_000
+    LMDB_MAP_SIZE = 1_000_000
```

Comment:
Unsure about using the smaller - I think the original large value is needed...see https://discordapp.com/channels/411401661714792449/411401661714792451/750728568509562931. @tuxxy ?